scripts/e2e-using-tekton/README.md

```diff
@@ -0,0 +1,25 @@
+# End-to-end tests using Tekton
+
+## About
+Run operator-sdk end-to-end tests written in ./test/e2e/ on a cluster with OpenShift GitOps installed. 
+
+## Steps 
+
+1. Install OpenShift GitOps on your cluster
+
+2. Create the following namespace to run your Pipelines.
+
+```
+oc new-project gitpos-e2e-test
+``` 
+
+3. Run the tests by 
+
+```
+oc apply -f e2e-pipeline.yaml
```

Comment:
It fails.
```
$ oc apply -f scripts/e2e-using-tekton/e2e-pipeline.yaml 
clusterrolebinding.rbac.authorization.k8s.io/pipeline-sa-admin unchanged
pipeline.tekton.dev/gitops-osdk-e2e created
unable to recognize "scripts/e2e-using-tekton/e2e-pipeline.yaml": no matches for kind "Task" in version ""
error when retrieving current configuration of:
Resource: "tekton.dev/v1beta1, Resource=pipelineruns", GroupVersionKind: "tekton.dev/v1beta1, Kind=PipelineRun"
Name: "", Namespace: "gitpos-e2e-test"
from server for: "scripts/e2e-using-tekton/e2e-pipeline.yaml": resource name may not be empty
```

Reply:
Sorry, had incorrectly assembled yamls in one file. Fixed it!